core/templates/pages/admin-page/admin-page.module.ts

```diff
@@ -103,6 +104,11 @@ import { SmartRouterModule } from 'hybrid-router-module-provider';
       provide: HAMMER_GESTURE_CONFIG,
       useClass: MyHammerConfig
     },
+    {
+      provide: ErrorHandler,
+      useClass: FirebaseErrorFilterHandler,
```

Comment:
Shouldn't these and other pages be using AppErrorHandler instead?

Reply:
`FirebaseErrorFilterHandler` inherits `AppErrorHandler`. Hence, FirebaseErrorFilterHandler is a `AppErrorHandler` but the name doesnot represent that. So, I changed the name to `AppErrorHandlerWithFirebaseErrorFilter`.

Reply:
Ah ok, yup that's fine. But should all pages be using this (with the Firebase error filter)? If so then should we just bake the filter into the default AppErrorHandler and not have a special subclass for firebase?

(My earlier suggestion re subclassing was based on the assumption that only app.module.ts needed the firebase stuff, but this is something I don't actually know the answer to.)

Reply:
Every page needs the firebase stuff. As `AngularFireAuthModule` is used in shared components module.

Reply:
Ok, so shall we just have a single AppErrorComponent that includes the firebase stuff? If it will be used everywhere then I don't see the need for an extra subclass.

Reply:
Done